src/Core/Tool/Authorizer/TagAuthorizer.php

```diff
@@ -78,7 +78,7 @@ public function isAllowed(Entity $entity, $privilege)
 			return true;
 		}
 
-		if ($privilege === 'search') {
+		if ($user->getId() and $privilege === 'search') {
```

Comment:
for tag I'm realizing we need the search. We just need to avoid leaking private tags

Reply:
(we need to see tags as annon)